### PR TITLE
Feature/sif relative path

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- Makes sifs relative to proxy bin

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: lib/MIP/Recipes/Install/Singularity.pm [hunk 1]
```diff
@@ -151,6 +152,11 @@ sub install_singularity_containers {
         $container_href->{program_bind_paths} = [];
 
         my $container_path = catfile( $container_dir_path, $container . q{.sif} );
```
Comment (jemten) [inline lib/MIP/Recipes/Install/Singularity.pm]: Still needed?

Comment (henrikstranneheim) [reply lib/MIP/Recipes/Install/Singularity.pm]: Yes, I think so to download the sif to the selected conda env by the user in mip install. Then we make the relative link. 
It looks like this:
```
singularity pull --force /home/henrik.stranneheim/miniconda3/envs/D_mip8.0_191019_hs/share/containers/cnvnator.sif
```
and
```
echo -n "singularity exec \"\$CONDA_ENV_DIR\"/share/containers/cnvnator.sif
```

**Comment by henrikstranneheim** [comment]
About the CLI - you are quite right I will remove the CLI option and lock this to a specific conda env to enable portability.

File: lib/MIP/Recipes/Install/Singularity.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Recipes/Install/Singularity.pm]: Don't you need/want to remove the `if ( not $container_dir_path ) {` above and have container_dir_path to always be `catdir( $conda_env_path, qw{ share containers } )`?

Comment (henrikstranneheim) [reply lib/MIP/Recipes/Install/Singularity.pm]: yes, you are right now that it is constant there is no need for the if condition.

**Comment by henrikstranneheim** [comment]
@jemten How about now?

**Pull request merged**